docs/README.adoc

```diff
@@ -1,34 +1,111 @@
-= Aktualizr reference docs
+= OTA Connect Developer Guide
 
-This directory contains developer-focused documentation on various specialized aktualizr topics.
+// MC: NOTE ABOUT TOC 
+// For the time being, this nav is mirrored from the docsite TOC two levels down in
+// 'aktualizr/docs/ota-client-guide/modules/ROOT/nav.adoc'. Need to sync these
+// two TOCS by hand until we can find a better solution. However "nav.adoc" should always be the
+// source of truth.
```

Comment:
Is this still true? If so, can we share the content somehow by having one doc reference the other?

Reply:
Will initually I thought it would be nice to have a a TOC on the 1st level of the docs dir. But I relize this isn't really necessary. If you look at the [README](https://github.com/advancedtelematic/aktualizr/tree/docs/OTA-2683/migrate-docs-antora/docs/README.adoc) now, I just link directly to the toc.

However, I have a new problem in that the links in "nav.adoc" don't work locally/Gitub since many location-related variables are only resolved at build-time. So I need add an if/then statement to show a nav that works locally/Github.